src/main/java/org/cactoos/io/UncheckedBytes.java

```diff
@@ -79,7 +80,7 @@ public UncheckedBytes(final Bytes bts,
     public byte[] asBytes() {
         byte[] data;
         try {
-            data = this.bytes.asBytes();
+            data = new IoCheckedScalar<>(this.bytes::asBytes).value();
```

Comment:
@pbenety The whole this class can be refactored now: 
- field `fallback` should have another type: `Func<Exception, byte[]>` (it will affect some of ctors as well).
- `catch` block should catch `Exception` instead of `IOException` (as `asBytes` throws Exception now)

You can do it yourself or just add a puzzle (it's explained [here](https://www.yegor256.com/2017/04/05/pdd-in-action.html) how it works and as well you can find examples in the project).